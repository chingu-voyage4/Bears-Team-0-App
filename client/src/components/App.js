import React, { Component } from 'react';
import { Route } from 'react-router-dom';
import Layout from './Layout/Layout';
import Landing from './Landing/Landing';
import Team from './Pages/Team';
import About from './Pages/About';
import Contact from './Pages/Contact';
import Dashboard from './Dashboard/Dashboard';
import MakeQuiz from './MakeQuiz/MakeQuiz';
import TakeQuiz from './TakeQuiz/TakeQuiz';
import Results from './Results/Results';
import Analysis from './Analysis/Analysis';
import Privacy from './Pages/Privacy';

class App extends Component {
  render() {
    return (
      <div className="app">
        <Layout>
          <Route exact path="/" component={Landing} />
          <Route exact path="/team" component={Team} />
          <Route exact path="/about" component={About} />
          <Route exact path="/privacy" component={Privacy} />
          <Route exact path="/contact" component={Contact} />
          <Route exact path="/dashboard" component={Dashboard} />
          <Route exact path="/makequiz" component={MakeQuiz} />
          <Route exact path="/results" component={Results} />
          <Route exact path="/analysis" component={Analysis} />
          <Route path="/takequiz/:id" component={TakeQuiz} />
        </Layout>
      </div>
    );
  }
}

export default App;
