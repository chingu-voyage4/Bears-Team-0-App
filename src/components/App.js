import React, { Component } from 'react';
import {Route} from "react-router-dom";
import Landing from "./Landing/Landing";
import AllQuizzes from './AllQuizzes/AllQuizzes'


class App extends Component {
  render() {
    return (
      <div className="App">
        <Route exact path="/" component={Landing}/>
        <Route exact path="/all" component={AllQuizzes} />
      </div>
    );
  }
}

export default App;