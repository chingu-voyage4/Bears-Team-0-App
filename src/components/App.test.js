import React from "react";
import ReactDOM from "react-dom";
import { MemoryRouter } from "react-router";
import App from "./App";

const app = (
  <MemoryRouter>
    <App />
  </MemoryRouter>
);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(app, div);
  ReactDOM.unmountComponentAtNode(div);
});
