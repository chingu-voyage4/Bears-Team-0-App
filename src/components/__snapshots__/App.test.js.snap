// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Layout>
        <Route
                component={[Function]}
                exact={true}
                path="/"
        />
        <Route
                component={[Function]}
                exact={true}
                path="/all"
        />
</Layout>,
      "className": "App",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Route
            component={[Function]}
            exact={true}
            path="/"
/>,
          <Route
            component={[Function]}
            exact={true}
            path="/all"
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "component": [Function],
            "exact": true,
            "path": "/all",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Layout>
          <Route
                    component={[Function]}
                    exact={true}
                    path="/"
          />
          <Route
                    component={[Function]}
                    exact={true}
                    path="/all"
          />
</Layout>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Route
              component={[Function]}
              exact={true}
              path="/"
/>,
            <Route
              component={[Function]}
              exact={true}
              path="/all"
/>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "component": [Function],
              "exact": true,
              "path": "/all",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
